cmip6_downscaling/workflows/bcsd_params.json

```diff
@@ -3,6 +3,20 @@
   "gcm": "MIROC6",
   "scenario": "ssp370",
   "variable": "tasmax",
+  "config.intermediate.uri": "az://flow-outputs/testing_intermediates",
+  "config.results.uri": "az://flow-outputs/testing_results",
+  "config.temporary.uri": "az://flow-outputs/temporary",
+  "config.intermediate.storage_options": {
+    "connection_string": "$AZURE_STORAGE_CONNECTION_STRING"
+  },
+  "config.results.storage_options": {
+    "connection_string": "$AZURE_STORAGE_CONNECTION_STRING"
+  },
+  "config.temporary.storage_options": {
+    "connection_string": "$AZURE_STORAGE_CONNECTION_STRING"
+  },
+  "runtime": "runtime",
+  "cleanup_flag": "cleanup_flag",
```

Comment:
question: can we specify a dict of config options that are arbitrarily unpacked during the flow run? This would mean just one key (`"config": {'intermediate.uri': ...}`. I would think that is a more flexible pattern.